rust/datafusion/Cargo.toml

```diff
@@ -56,6 +56,7 @@ num_cpus = "1.13.0"
 chrono = "0.4"
 async-trait = "0.1.41"
 futures = "0.3"
+pin-project-lite= "^0.1.1"
```

Comment:
Note that this does not add any new actual dependencies (pin_project_lite is already required indirectly via tokio)

Reply:
You did well. This IMO is the correct way to approach this. All relevant async projects use this lib, and even std uses a variation of it AFAIK.